recipe/meta.yaml

```diff
@@ -180,6 +179,8 @@ outputs:
         - libnsl  # [linux]
         - libuuid  # [linux]
         - libxcrypt  # [linux]
+        - xorg-libx11               # [linux]
+        - xorg-xorgproto            # [linux]
```

Comment:
@xhochy, these are supposed to be added to `ignore_run_exports` in the python recipe to avoid x11 being pulled into every python environment.

Reply:
The aarch tests were failing due to missing libxcb before. Probably `xorg-libx11` can be reduced to that.

Reply:
Confirmed in #706 that `libxcb` itself is not enough, so we need `xorg-libx11`, but ignore its run-export; `xorg-xorgproto` has no run-export.